server/spinwick.go

```diff
@@ -65,6 +89,103 @@ func (s *Server) handleCreateSpinWick(pr *model.PullRequest, size string, withLi
 			s.logPrettyErrorToMattermost("[ SpinWick ] Creation Failed", pr, request.Error, additionalFields)
 		}
 	}
+
+}
+
+func (s *Server) createKubeSpinWick(pr *model.PullRequest) *spinwick.Request {
+	request := &spinwick.Request{
+		InstallationID: "n/a",
+		Error:          nil,
+		ReportError:    false,
+		Aborted:        false,
+	}
+
+	logger := log.WithField("PR", pr.RepoName+"/!"+string(pr.Number))
```

Comment:
i think we should create the client using the code i shared.

Reply:
I have implemented your code. The logger is still needed to create a KubeClient